README.md

```diff
@@ -1,7 +1,17 @@
 # Awesome Dev Articles [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
-A curated list of articles
+A curated list of articles created by the ESSS Dev team
 
 Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 ## Articles
+
+* [Steps to better code](https://medium.com/@isaaclyman/steps-to-better-code-e6c3cce0c7f9) - Isaac Lyman
```

Comment:
How about sorting this by title or date?

Reply:
It's sorted by complexity. My intention was to make it easier for newcomers to pick what to read. Maybe we could sort by title and add sections "Basic" and "Advanced"

Reply:
Sounds good.